resources/datasets/hmda/Drakefile

```diff
@@ -60,23 +60,26 @@ gaz_counties.zip <- [-timecheck]
 gaz_counties.csv <- gaz_counties.zip
   unzip -p $INPUT | iconv -f ISO-8859-2 -t UTF-8 | perl -lpe 's/\s*$//;s/\t/,/g' > $OUTPUT
 
-hmda_lar_2013.zip, hmda_lar_2012.zip, hmda_lar_2011.zip, hmda_lar_2010.zip, hmda_lar_2009.zip, hmda_lar_2008.zip, hmda_lar_2007.zip <- [-timecheck]
-  curl http://www.ffiec.gov/hmdarawdata/LAR/National/2013HMDALAR%20-%20National.zip > $OUTPUT0
-  curl http://www.ffiec.gov/hmdarawdata/LAR/National/2012HMDALAR%20-%20National.zip > $OUTPUT1
-  curl http://www.ffiec.gov/hmdarawdata/LAR/National/2011HMDALAR%20-%20National.zip > $OUTPUT2
-  curl http://www.ffiec.gov/hmdarawdata/LAR/National/2010HMDALAR%20-%20National.zip > $OUTPUT3
-  curl http://www.ffiec.gov/hmdarawdata/LAR/National/2009HMDALAR%20-%20National.zip > $OUTPUT4
-  curl http://www.ffiec.gov/hmdarawdata/LAR/National/2008HMDALAR%20-%20National.zip > $OUTPUT5
-  curl http://www.ffiec.gov/hmdarawdata/LAR/National/2007HMDALAR%20-%20National.zip > $OUTPUT6
-
+hmda_lar_2014.zip, hmda_lar_2013.zip, hmda_lar_2012.zip, hmda_lar_2011.zip, hmda_lar_2010.zip, hmda_lar_2009.zip, hmda_lar_2008.zip, hmda_lar_2007.zip <- [-timecheck]
+  curl http://www.ffiec.gov/hmdarawdata/LAR/National/2014HMDALAR%20-%20National.zip > $OUTPUT0
+  curl http://www.ffiec.gov/hmdarawdata/LAR/National/2013HMDALAR%20-%20National.zip > $OUTPUT1
+  curl http://www.ffiec.gov/hmdarawdata/LAR/National/2012HMDALAR%20-%20National.zip > $OUTPUT2
+  curl http://www.ffiec.gov/hmdarawdata/LAR/National/2011HMDALAR%20-%20National.zip > $OUTPUT3
+  curl http://www.ffiec.gov/hmdarawdata/LAR/National/2010HMDALAR%20-%20National.zip > $OUTPUT4
+  curl http://www.ffiec.gov/hmdarawdata/LAR/National/2009HMDALAR%20-%20National.zip > $OUTPUT5
+  curl http://www.ffiec.gov/hmdarawdata/LAR/National/2008HMDALAR%20-%20National.zip > $OUTPUT6
+  curl http://www.ffiec.gov/hmdarawdata/LAR/National/2007HMDALAR%20-%20National.zip > $OUTPUT7
+
+hmda_lar_all_2014_no_fips.csv <- hmda_lar_2014.zip [method:add_headers_and_unzip]
 hmda_lar_all_2013_no_fips.csv <- hmda_lar_2013.zip [method:add_headers_and_unzip]
 hmda_lar_all_2012_no_fips.csv <- hmda_lar_2012.zip [method:add_headers_and_unzip]
 hmda_lar_all_2011_no_fips.csv <- hmda_lar_2011.zip [method:add_headers_and_unzip]
 hmda_lar_all_2010_no_fips.csv <- hmda_lar_2010.zip [method:add_headers_and_unzip]
 hmda_lar_all_2009_no_fips.csv <- hmda_lar_2009.zip [method:add_headers_and_unzip]
 hmda_lar_all_2008_no_fips.csv <- hmda_lar_2008.zip [method:add_headers_and_unzip]
-hmda_lar_all_2007_no_fips.csv <- hmda_lar_2007.zip [method:add_headers_and_7unzip]
```

Comment:
`7unzip` is not a typo, there's actually an `add_headers_and_7unzip` function.  Is that function no longer working for you using the 2007 ffiec zipfile?